src/redisearch_rs/trie_rs/src/trie.rs

```diff
@@ -79,19 +79,20 @@ impl<Data> TrieMap<Data> {
             // After removing the child, we attempt to merge the child into the root.
             root.merge_child_if_possible();
             data
-        }
+        };
+        data
```

Comment:
Is change related to the replacement of c_char with u8?

Reply:
This one slipped through the cracks, removed.